File: Samples/EasyImGui.Samples/MultiWindow/Form1.cs
using EasyImGui;
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Threading;
using System.Threading.Tasks;
using System.Windows.Forms;

namespace MultiWindow
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }

        public bool ShowImguiDemo = false;
        MultiImGuiForm MultiForm = null;

        private void Form1_Load(object sender, EventArgs e)
        {
            
            MultiForm = new MultiImGuiForm();
            MultiForm.Render += delegate
            {
                if (!MultiForm.Draw)
                    return false;

                DearImguiSharp.ImGui.Begin("Another Window in C#", ref MultiForm.Draw, (int)DearImguiSharp.ImGuiWindowFlags.None);
                DearImguiSharp.ImGui.Text("Hello from another window!");

                if (DearImguiSharp.ImGui.Button("ImguiDemo", new DearImguiSharp.ImVec2() { X = 200, Y = 20 }))
                    ShowImguiDemo = !ShowImguiDemo;

                if (ShowImguiDemo)
                    DearImguiSharp.ImGui.ShowDemoWindow(ref ShowImguiDemo);

                DearImguiSharp.ImGui.End();
                return true;
            };

            MultiForm.Show();
        }

        private void button1_Click(object sender, EventArgs e)
        {
            if (MultiForm != null)
                MultiForm.Draw = true;
        }

        private void button2_Click(object sender, EventArgs e)
        {
            if (MultiForm != null)
                MultiForm.Draw = false;
        }
    }
}

File: Samples/EasyImGui.Samples/MultiWindowVB/Form1.vb
Imports EasyImGui

Public Class Form1

    Public ShowImguiDemo As Boolean = False
    Private MultiForm As MultiImGuiForm = Nothing

    Private Sub Form1_Load(sender As Object, e As EventArgs) Handles MyBase.Load
        MultiForm = New MultiImGuiForm()
        AddHandler MultiForm.Render, Function()
                                         If Not MultiForm.Draw Then Return False

                                         DearImguiSharp.ImGui.Begin("Another Window in C#", MultiForm.Draw, CInt(DearImguiSharp.ImGuiWindowFlags.None))
                                         DearImguiSharp.ImGui.Text("Hello from another window!")

                                         If DearImguiSharp.ImGui.Button("ImguiDemo", New DearImguiSharp.ImVec2() With {
                                             .X = 200,
                                             .Y = 20
                                         }) Then ShowImguiDemo = Not ShowImguiDemo

                                         If ShowImguiDemo Then DearImguiSharp.ImGui.ShowDemoWindow(ShowImguiDemo)

                                         DearImguiSharp.ImGui.[End]()
                                         Return True
                                     End Function

        MultiForm.Show()
    End Sub

    Private Sub button1_Click(sender As Object, e As EventArgs) Handles button1.Click
        If MultiForm IsNot Nothing Then MultiForm.Draw = True
    End Sub

    Private Sub button2_Click(sender As Object, e As EventArgs) Handles button2.Click
        If MultiForm IsNot Nothing Then MultiForm.Draw = False
    End Sub

End Class
